types/wlr_xdg_output_v1.c

```diff
@@ -132,6 +132,10 @@ static void output_manager_handle_get_xdg_output(struct wl_client *client,
 	}
 
 	output_send_details(xdg_output, xdg_output_resource);
+
+	if (version => OUTPUT_DONE_DEPRECATED_SINCE_VERSION) {
```

Comment:
Hmm, actually one issue is that this will broadcast the done event for all clients. We only need to schedule it for `output_resource`.

Reply:
Maybe we could just immediately send `wl_output.done` and leave the atomicity as a TODO.